### DISCUSSION TITLE
message passing to pybricks for ev3

### DISCUSSION THREAD

**Discussion by bblais** [pybricks/support#641] [Q&A]
I am new to pybricks, but have used ev3dev in the past.  I was wondering if there is a way to get message passing to work between, say, a laptop and the ev3?  Since pybricks uses micropython, the libraries seem limited (no numpy, for example), so I thought I could offload the hard stuff and just have my laptop script control the ev3 -- sending instructions and sensor readings back and forth -- but I don't see any examples of this anywhere.  thanks!

**Comment by laurensvalk** [comment]
You can try this example: https://github.com/pybricks/pybricks-api/tree/master/examples/ev3/bluetooth_pc

It makes your PC behave just like an EV3 client, so communication works just like it does between two EV3s.
